### PR TITLE
WDL workflow for ichorCNA copy number calling tool

### PR DESCRIPTION
**Pull request by prisnir**

### CODE REVIEW

File: workflow/ichorCNA.wdl [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  sampleName=sampleName,
+  modules=modules,
+  mem=mem,
+  bamWig=runReadCounter.bamWig
+
+ }
+}
+
+
+task copyOverBam {
```
Comment (pruzanov) [inline workflow/ichorCNA.wdl]: I am not sure why this is here, is it used anywhere?

File: workflow/ichorCNA.wdl [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    --centromere $ICHORCNA_ROOT/lib/R/ichorCNA/extdata/GRCh37.p13_centromere_UCSC-gapTable.txt \
+    --normalPanel $ICHORCNA_ROOT/lib/R/ichorCNA/extdata/HD_ULP_PoN_1Mb_median_normAutosome_mapScoreFiltered_median.rds \
+    --includeHOMD ~{true="True" false="False" includeHOMD} \
+    --chrs "c(1:22, 'X')" \
+    --chrTrain "c(1:22)" \
+    --estimateNormal True \
+    --estimatePloidy True \
+    --estimateScPrevalence True \
+    --scStates "c(1, 3)" \
+    --txnE 0.9999 \
```
Comment (pruzanov) [inline workflow/ichorCNA.wdl]: these may be parametrized

**Comment by mlaszloffy** [comment]
retest this please

**Comment by mlaszloffy** [comment]
retest this please

**Comment by mlaszloffy** [comment]
retest this please

**Comment by mlaszloffy** [comment]
retest this please

**Comment by mlaszloffy** [comment]
retest this please

**Comment by mlaszloffy** [comment]
retest this please

File: workflow/ichorCNA.wdl (hunk 1 above)
Comment (mlaszloffy) [reply workflow/ichorCNA.wdl]: This task has been removed.

File: workflow/ichorCNA.wdl (hunk 2 above)
Comment (mlaszloffy) [reply workflow/ichorCNA.wdl]: Parametrized all ichorCNA options.

**Pull request closed without merging**